model-optimizer/extensions/front/tf/identityN_to_identity.py

```diff
@@ -41,12 +41,19 @@ def replace_identityN(node: Node):
         dtypes = node.data_types
 
         for idx, port in node.in_ports().items():
-            assert node.is_out_port_connected(idx), 'IdentityN {} has inconsistent input and output ports'.format(name)
+            if not node.is_in_port_connected(idx) or not node.is_out_port_connected(idx):
```

Comment:
Please, leave a comment here describing the case when the semantic may be broken.